lib/gemini.js

```diff
@@ -207,7 +207,7 @@ function applyReporter(runner, reporter) {
             throw e;
         }
 
-        return reporter(runner, reporterPath);
+        return reporter(runner, reporterOptions);
```

Comment:
I don't think it can be just ommited. Reporter variable is replaced with reporter function and the reporter.options should not be available anymore, or is it?